frame/examples/offchain-worker/src/lib.rs

```diff
@@ -242,19 +244,18 @@ pub mod pallet {
 		/// Submit new price to the list via unsigned transaction.
 		///
 		/// Works exactly like the `submit_price` function, but since we allow sending the
-		/// transaction without a signature, and hence without paying any fees,
-		/// we need a way to make sure that only some transactions are accepted.
-		/// This function can be called only once every `T::UnsignedInterval` blocks.
```

Comment:
If you find a linter that will prohibit people from immaturely wrapping their lines early, I will indefinitely be grateful of you.

Reply:
💯

Reply:
> If you find a linter that will prohibit people from immaturely wrapping their lines early, I will indefinitely be grateful of you.

💯